pkgs/coverage/bin/test_with_coverage.dart

```diff
@@ -218,8 +208,9 @@ Future<void> main(List<String> arguments) async {
   );
   final serviceUri = await serviceUriCompleter.future;
 
-  final scopes =
-      flags.scopeOutput.isEmpty ? [flags.packageName] : flags.scopeOutput;
+  final scopes = flags.scopeOutput.isEmpty
```

Comment:
This completely removes the packageName rather than deprecating it right? Better to fully remove it then if it's not supported anymore?

Reply:
That would be a breaking change.